docs/user/QUICKSTART.md

```diff
@@ -1,4 +1,4 @@
-## Introduction
+# Quick Start
```

Comment:
Let's keep the name `Quickstart` to stay consistent.

cref: https://en.wikipedia.org/wiki/Quickstart_guide